notebooks/249-fastcomposer-image-generation/249-fastcomposer-image-generation.ipynb

```diff
@@ -0,0 +1,1327 @@
+{
```

Comment:
<p>please add table of contents</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Added